addons/packages/secretgen-controller/0.5.0/bundle/config/values.yaml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+namespace: secretgen-controller
+secretgenController:
+  namespace: null
```

Comment:
Why do we need to have namespace in both line 4 and line 6. Other packages did this because of backward compatibility. Since it's a newly added package, we should be fine with either option.